tests/ternaries/__snapshots__/jsfmt.spec.js.snap

```diff
@@ -255,6 +401,33 @@ a =>
 
 `;
 
+exports[`parenthesis.js 4`] = `
+debug ? this.state.isVisible ? "partially visible" : "hidden" : null;
+debug ? this.state.isVisible && somethingComplex ? "partially visible" : "hidden" : null;
+
+a => a ? () => {a} : () => {a}
+a => a ? a : a
+a => a ? aasdasdasdasdasdasdaaasdasdasdasdasdasdasdasdasdasdasdasdasdaaaaaa : a
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+debug ? (this.state.isVisible ? "partially visible" : "hidden") : null;
+debug
+	? this.state.isVisible && somethingComplex ? "partially visible" : "hidden"
+	: null;
+
+a =>
+	a
+		? () => {
+				a;
+			}
+		: () => {
+				a;
+			};
+a => (a ? a : a);
+a =>
+	a ? aasdasdasdasdasdasdaaasdasdasdasdasdasdasdasdasdasdasdasdasdaaaaaa : a;
```

Comment:
Is there just one space at the start of this line?

Reply:
1 tab, it's the current behavior.

You can see 37b2ece for actual changes related to this PR.